playbooks/scale.yml

```
@@ -5,22 +5,8 @@
 - name: Gather facts
   import_playbook: facts.yml
 
-- name: Generate the etcd certificates beforehand
-  hosts: etcd:kube_control_plane
-  gather_facts: false
-  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
-  environment: "{{ proxy_disable_env }}"
-  roles:
-    - { role: kubespray-defaults }
-    - role: etcd
-      tags: etcd
-      vars:
-        etcd_cluster_setup: false
-        etcd_events_cluster_setup: false
```

The only slight concern I have is this, which differs in install-etcd.yml.

However this only affect 'etcd', so it would only be a problem for using scale.yml with etcd nodes, which is not supported. I wonder if we have an easy way to fail early if something like this is attempted :thinking: 

Maybe that's out of scope for this though.

@tico88612 @ant31 thoughts ?

---

can we set the variables in the import?

---

The default value for etcd_cluster_setup is set to true. However, this can be changed to false by configuring the variables in scale.yaml. 

https://github.com/kubernetes-sigs/kubespray/blob/master/roles/etcd/defaults/main.yml#L6

@VannTen and @ant31 please review my changes.

---

Oh right, I don't why I thought we could not set vars on `import_playbook`.

In that case we'd define in both cases at import_playbook level and be done with it.

---

updated.